controller/src/main/java/org/jboss/as/controller/AttributeDefinition.java

```diff
@@ -812,6 +815,10 @@ public ModelNode getNoTextDescription(boolean forOperation) {
                 result.get(ModelDescriptionConstants.REQUIRES).add(required);
             }
         }
+        if (referenceRecorder != null) {
+            result.get(ModelDescriptionConstants.CAPABILITY_REFERENCE).set(referenceRecorder.getBaseRequirementName());
```

Comment:
Please ping @heiko-braun as to whether he'd like to see all 3 pieces of data here: requirement name, dependent name, whether dependent is dynamic. All that info might be overkill, but OTOH once we ship with a data structure we are stuck with it. My instinct is to report more.